app/views/user_mailer/tester.text.erb

```diff
@@ -0,0 +1,5 @@
+<%= _("Foreman audit summary") %>
+=================================
+<%= _("Foreman mail tester") %>
+<%= _("The email configurations are well set") %>
+<%= _("Return to Foreman:  %{foreman_url} ") % { :foreman_url => Setting[:foreman_url] } %>
```

Comment:
I'd prefer something like (both in HTML and TEXT):

```
Foreman test e-mail

This text confirms Foreman e-mail configuration.
```

No need to provide Foreman URL I think.